src/@next/Pagination/paginationUtils.ts

```diff
@@ -9,6 +11,14 @@ export type PaginationProps = {
   onPageChanged?: (currentPage: number) => void;
 };
 
+export type IconPaginationProps = {
+  label?: React.ReactNode;
+  hasNext?: boolean;
```

Comment:
can we use `disableNext` and `disablePrevious` with a default of `false`? `has` seems like it should not be there but what we want is for it to be disabled.